templates/lzv-rights/discrimination-lzv.jade

```diff
@@ -82,13 +97,15 @@ html
 						h2 {{score}}/{{answers.length}}
 						p {{resultText[score].description}}
 						button.quiz_button(@click='restart()') Спробувати ще
-		#rec68554931.r.t-rec.t-rec_pt_60.t-rec_pb_60.r_anim.r_showed(style='padding-top:60px;padding-bottom:60px; ;', data-record-type='106')
+		#rec68554931.r.t-rec.t-rec_pt_60.t-rec_pb_60.r_anim.r_showed(style='padding-top:50px;padding-bottom:5px; ;', data-record-type='106')
 			// T004
 			.t004
 				.t-container
 					.t-col.t-col_11
 						.t-text.t-text_md(field='text')
-							| Спецпроект реалізований в рамках проекту «Прискорення заходів з подолання ВІЛ-інфекції в Україні» (HealthLink) за підтримки Агентства США з міжнародного розвитку (USAID).
+							| Спецпроект реалізований в рамках проекту 
+							a(href="http://network.org.ua/ru/2018/06/06/v-ukrayini-rozpochato-masshtabnyj-proekt-z-podolannya-vil-snidu-healthlink/")«Прискорення заходів з подолання ВІЛ-інфекції в Україні» (HealthLink)
+							|  за підтримки Агентства США з міжнародного розвитку (USAID).
 		script(src='https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js')
 		script.
```

Comment:
It will be better to create a new `.js` file with this `Vue` application. Maybe even better is to create a separate application (using `vue-cli` e.g.), minify it and just link with `#quiz_application`, because it will also render `HTML` templates.
Same as below.